kernel.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+#DELME!
+
+
+# Install generic RPM build dependencies.
+dnf install -y fedora-packager fedpkg grubby ncurses-devel pesign rpm-build rpmdevtools 'dnf-command(builddep)'
+# Download the kernel package repository. This requires first increasing the buffer size that git uses.
+git config --global http.postBuffer 314572800
+fedpkg clone -a kernel
+cd kernel
+# This is the last commit for Linux 6.3.13 before it was rebased to Linux 6.4.4.
+git checkout e0412c4338e57bc8271c644b56a0a10f220b015c
```

Comment:
Any sense in creating a variable for the commit hash so it could be updated from the pipeline variables? or is this last commit a permanent fixture?

Reply:
This is a permanent fixture. I also expect that we will be drastically changing how we build and manage the Linux kernel later this year.